src/pages/Feed/FeedBtnList.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import * as F from './Feed.styles';
+import {
+  AiOutlineHeart,
+  AiOutlineStar,
+  AiFillHeart,
+  AiFillStar,
+} from 'react-icons/ai';
+import { FaRegComment } from 'react-icons/fa';
+import { GrShareOption } from 'react-icons/gr';
+
+const FeedBtnList = ({ likesNum, scrapNum, comments, reviews }) => {
+  const [likes, setLikes] = useState(false);
+  const [scrap, setScrap] = useState(false);
+  const [commentOn, setCommentOn] = useState(false);
+  const [comment, setComment] = useState('');
+  const [commentList, setCommentList] = useState([]);
+
+  const handleLikes = () => {
+    setLikes(!likes);
+  };
+
+  const handleStar = () => {
+    setScrap(!scrap);
+  };
+
+  const handleComment = () => {
+    setCommentOn(!commentOn);
+  };
+
+  const addCommenValue = e => {
+    setComment(e.target.value);
+  };
+
+  const addComment = e => {
+    if (e.key === 'Enter' && comment.length >= 1) {
+      setCommentList([...commentList, comment]);
+      setComment('');
+    }
+  };
+
+  return (
+    <F.CommentList>
+      <F.CommentDataList>
+        {reviews.map(
+          ({ nickname, profileImage, content, createdAt }, index) => {
+            return (
+              <F.CommentData key={index}>
+                <F.CommentUserImg src={profileImage} />
+                <F.CommentUserId>{nickname}</F.CommentUserId>
+                <F.UsersComment>{content}</F.UsersComment>
+              </F.CommentData>
+            );
+          }
+        )}
+        {commentList.length > 0 &&
+          commentList.map((data, index) => {
+            return (
+              <F.CommentData key={index}>
+                <F.CommentUserImg src="http://k.kakaocdn.net/dn/uCsLz/btrJmxO8ewM/uEhkd5isOAcF2HeTuwpFl1/img_640x640.jpg" />
+                <F.CommentUserId>97_bhh</F.CommentUserId>
+                <F.UsersComment>{data}</F.UsersComment>
+              </F.CommentData>
+            );
+          })}
+      </F.CommentDataList>
+      <F.BtnList>
+        <F.Btns>
+          <F.ContentBtn onClick={handleLikes}>
+            {likes === true ? <AiFillHeart /> : <AiOutlineHeart />}
```

Comment:
```suggestion
            {likes ? <AiFillHeart /> : <AiOutlineHeart />}
```
likes 값 자체가 boolean 값이기 때문에 위와 같이 구현해 주시는게 좋습니다!